src/Config/Main.hs

```diff
@@ -36,6 +38,15 @@ import UI.ComposeEditor.Keybindings
 import Types
 import Storage.Notmuch (getDatabasePath)
 
+sendmailPath :: FilePath
+sendmailPath = "/usr/sbin/sendmail"
+
+renderSendMail :: B.ByteString -> IO String
+renderSendMail m =
+        -- -t which extracts recipients from the mail
+        -- TODO: accept a ByteString as stdin
+        readProcess sendmailPath ["-t", "-v"] (C8.unpack m)
```

Comment:
`readProcess` can also block, throw exception. OK for now but file issue to address these problems.

Reply:
Yeah it seems you saw the comment on the calling site.

Reply:
See #201. I'll update the comments and repush.